### DISCUSSION TITLE
What is the direction (x, y and z) of MSD ?

### DISCUSSION THREAD

**Discussion by LiuHaoXuan0403** [ttadano/alamode#118] [Q&A]
Hi Dr. Tadano,
I try to use ALAMODE to calculate the mean square displacement (MSD). I can find three directions (x, y and z) of MSD in the file prefix.scph_msd. Can you guide me on whether the direction (x, y and z) of MSD is the basis vector of supercell or primitive cell.

**Comment by ttadano** [comment]
The x, y, and z directions are defined in the global Cartesian frame.

**Reply by zentenmiyata** [reply]
Dear Tadano-sensei,

 This is Miyata. Excuse me for interrupting.
I thought that the components in each direction of the MSD were amplitudes with respect to the μ direction defined by the crystal axes of the supercell, but are they output in the Cartesian coordinate system?
Alm explained that uμ(ℓκ) is the atomic displacement of the κth atom in the ℓth unit cell along the μ direction, so I thought the μ direction was defined with respect to the crystal axis.

So far, I have only discussed average values, so it is not a problem, but could you please tell me for the future?

As well as MSDs, are the xx, yy and zz of thermal conductivity similarly defined in a Cartesian coordinate system? For example, in a hexagonal system, does this mean that the a-axis is x, the c-axis is z and the b-axis requires a matrix transformation? Or is it only MSD that is defined in the Cartesian coordinate system? Are there any references that explain the definition of the coordinate system in an easy-to-understand manner, e.g. with pictures?

Best regards,
Miyata

**Reply by ttadano** [reply]
The directions of force constants, MSD, group velocity, and lattice thermal conductivity are all defined in the Cartesian frame (x,y,z). The crystal axes (a,b,c) are not used as a basis of the final outputs.

**Reply by zentenmiyata** [reply]
Dear Tadano-sensei,

 Thank you very much for the answer.

Best regards,
Miyata
